fashion-messaging-app-api/server.js

```diff
@@ -173,45 +172,103 @@ app.get('/recommendations', async (req, res) => {
       return res.status(401).json({ error: 'Unauthorized' });
     }
 
+    let recommendations = [];  
+    const frequentSearch = await SearchResult.findOne({
+      where: { 
+        userId: req.session.user.id,
+        searchTerm: {
+          [Op.not]: defaultQuery  
```

Comment:
I used this to query the database but exclude the default query "Fashion" to avoid repetition in my recommendation